src/test/java/hudson/remoting/TrafficAnalyzer.java

```diff
@@ -21,23 +21,24 @@ public static void main(String[] args) throws Exception {
         File f = new File("/home/kohsuke/ws/hudson/investigations/javafx-windows-hang/out.log");
         DataInputStream fin = new DataInputStream(new FileInputStream(f));
```

Comment:
Shouldn't the read also be included in a try with resources?
```suggestion
        try (DataInputStream fin = new DataInputStream(new FileInputStream(f))) {
```